docs/AboutUs.md

```diff
@@ -5,7 +5,14 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at our emails
+```
+gordonyit25@u.nus.edu
+e0564779@u.nus.edu
+e0559731@u.nus.edu
+chunweilee.99@u.nus.edu
+ng.xiangjun99@u.nus.edu
+```
```

Comment:
Would it be better if we just provide one email? I think it is easier for the users since they don't need to choose from 5 emails and also it is easier for us because we only need to monitor one email. What do you think?

Reply:
Maybe we can remove the emails from the start of the page, then include links to our email in our own individual sections?

Reply:
How about we discuss about the including of link in the future if we want after merging? So one person updating is enough since doing this now might require everyone to commit the changes to their pr.